crs_application/config/main/motion_planning/MP_config.yaml

```diff
@@ -11,16 +11,19 @@ general:
   max_joint_vel: 0.3
   max_joint_vel_mult: 1.80
   max_surface_dist: 0.15
+  max_rotation_rate: 3.0
   max_joint_acc: 0.8
   add_approach_and_retreat: true
-  minimum_raster_length: 4
+  minimum_raster_length: 5
+  reachable_radius: 1.35
   trajopt_verbose_output: false
   combine_strips: true
   global_descartes: true
 descartes:
-  axial_step: 0.4
+  axial_step: 0.1
   collision_safety_margin: 0.00
   additional_tool_offset: [0.0, 0.0, 0.004, 0.0, 0.0, 0.0]
+  joint_min_vals: [-0.1, -0.1, -0.4, -0.1, 0.1, -0.1]
```

Comment:
How are these used?

Reply:
These are used to check for valid joint states in descartes and are minimum allowable absolute values for each joint, so anything negative means that the corresponding joint is not restricted at all. The only thing this particular parameter does right now is ensure that wrist_2_joint has a value outside of -0.1 to 0.1 to avoid wrist singularities.